src/wolfkrow/core/tasks/concatenate_quicktime.py

```diff
@@ -0,0 +1,106 @@
+""" Module implementing the FileCopy task.
```

Comment:
doc string

Reply:
Gahh! A bunch of copy/paste errors in here. Do a full pass through this and update all the references to the copied task definition.